pkg/controller/provider/web/graph/model/models_gen.go

```diff
@@ -0,0 +1,10 @@
+// Code generated by github.com/99designs/gqlgen, DO NOT EDIT.
+
+package model
+
+type VsphereProvider struct {
+	ID      string `json:"id"`
+	Name    string `json:"name"`
+	Kind    string `json:"kind"`
+	Product string `json:"product"`
```

Comment:
What is a product?

Reply:
That's a reminiscent of an VMware object in Opal schema. 
Never mind, I'll remove it.